pyproject.toml

```diff
@@ -1,3 +1,67 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "nwbinspector"
+version = "0.6.0"
```

Comment:
not sure if the version was meant to be updated in this PR?

Reply:
It was. Just in case. (using a different build backend is a fundamental change; not per se one expected to break back compatibility, but it's fairly significant)

Maybe after the remaining big hard deprecations next March you might want to do an official v1.0.0 release

Reply:
That makes sense. And yes an official release after the big hard deprecations sounds like a good plan.